include/drivers/mm/mm_drv_intel_adsp_mtl_tlb.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Author: Marcin Szkudlinski <marcin.szkudlinski@linux.intel.com>
+ *
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_INTEL_ADSP_MTL_TLB
+#define ZEPHYR_INCLUDE_DRIVERS_INTEL_ADSP_MTL_TLB
+
+/* device id from the device tree */
+#define DT_DRV_COMPAT_ADSP_TLB intel_adsp_tlb
+
+/* size of TLB table */
+#define TLB_SIZE DT_REG_SIZE_BY_IDX(DT_INST(0, DT_DRV_COMPAT_ADSP_TLB), 0)
+
+/* numer of L2 memory pages */
+#define L2_SRAM_PAGES_NUM (L2_SRAM_SIZE / CONFIG_MM_DRV_PAGE_SIZE)

```

Comment:
Not sure if adding unused - in this patch - definitions helps with clean-up. Maybe defer adding those to when they are needed?

Reply:
changed